data/src/main/scala/ch.epfl.scala.index.data/bintray/BintrayPipeline.scala

```diff
@@ -71,12 +65,6 @@ object BintrayPipeline {
       println("github done")
     }
 
-    def live(): Unit = {
-
-      val liveStep = new SaveLiveData(paths)
-      liveStep.run()
-    }
```

Comment:
This step is now unnecessary because “live” data is saved to the `index` repository when users submit the edit form.